packages/react-components/src/components/TogetherModeOverlay.tsx

```diff
@@ -0,0 +1,275 @@
+
+      const participantsWithVideoAvailable = allParticipants.filter(
+        (p) => p.videoStream?.isAvailable && togetherModeSeatPositions[p.userId]
+      );
+      const updatedSignals = participantsWithVideoAvailable.reduce(
+        (accumulator: { [key: string]: TogetherModeParticipantStatus }, p: VideoGalleryLocalParticipant) => {
+          const { userId, reaction, raisedHand, spotlight, isMuted, displayName } = p;
+          const seatingPosition = togetherModeSeatPositions[userId];
+          if (seatingPosition) {
+            accumulator[userId] = {
+              id: userId,
+              reaction,
+              isHandRaised: !!raisedHand,
+              isSpotlighted: !!spotlight,
+              isMuted,
+              displayName: displayName || locale.strings.videoGallery.displayNamePlaceholder,
+              showDisplayName: !!(spotlight || raisedHand || reaction || hoveredParticipantID === userId),
+              scaledSize: calculateScaledSize(seatingPosition.width, seatingPosition.height),
+              seatPositionStyle: setTogetherModeSeatPositionStyle(seatingPosition)
+            };
+          }
+          return accumulator;
+        },
+        {}
+      );
+
+      const participantsNotInTogetherModeStream = Object.keys(togetherModeParticipantStatus).filter(
+        (id) => !togetherModeParticipantStatus[id]
+      );
+
+      setTogetherModeParticipantStatus((prevSignals) => {
+        const newSignals = { ...prevSignals, ...updatedSignals };
+        participantsNotInTogetherModeStream.forEach((id) => {
+          delete newSignals[id];
+        });
+
+        const hasChanges = Object.keys(newSignals).some(
+          (key) => JSON.stringify(newSignals[key]) !== JSON.stringify(prevSignals[key])
+        );
+
+        return hasChanges ? newSignals : prevSignals;
+      });
+    }, [
+      remoteParticipants,
+      localParticipant,
+      togetherModeParticipantStatus,
+      togetherModeSeatPositions,
+      locale.strings.videoGallery.displayNamePlaceholder,
+      hoveredParticipantID
+    ]);
+
+    return (
+      <div style={{ position: 'absolute', width: '100%', height: '100%' }}>
+        {Object.values(togetherModeParticipantStatus).map((participantStatus) => (
+          <div
+            key={participantStatus.id}
+            style={{
+              ...getTogetherModeParticipantOverlayStyle(participantStatus.seatPositionStyle),
+              border: '1px solid yellow'
+            }}
+            onMouseEnter={() => setHoveredParticipantID(participantStatus.id)}
+            onMouseLeave={() => setHoveredParticipantID('')}
+          >
+            <div>
+              {participantStatus.reaction?.reactionType && (
+                <div
+                  style={moveAnimationStyles(
+                    parseFloat(participantStatus.seatPositionStyle.seatPosition.height) * 0.5,
+                    parseFloat(participantStatus.seatPositionStyle.seatPosition.height) * 0.35
```

Comment:
Can we get some constants to define these numbers? just so we can know what they do in the future if we need to update them